src/client/components/RoomSelector.jsx

```diff
@@ -28,10 +28,11 @@ const RoomSelector = ({socket, joinRoom, className}) => {
     ev.preventDefault();
     const formData = new FormData(ev.target);
     const {roomName, isPrivate} = Object.fromEntries(formData.entries());
-    if(!roomName){
+    const validatedRoomName = roomName.trim().slice(0, 50);
```

Comment:
While it makes sense to slice in the backend, it's already impossible to get anything longer than 50 in the frontend without messing with the code, so you can remove slicing here and for the `userName`.

Reply:
Agree! Also because of refactor and required field - it was possible to add a regex that prevents user to type name that consists only with spaces. And added `title` attribute that is show as part of error message in those cases.